linux/LinuxProcessTable.c

```diff
@@ -93,6 +93,34 @@ static inline uint64_t fast_strtoull_dec(char** str, int maxlen) {
    return result;
 }
 
+static long long fast_strtoll_dec(char** str, int maxlen) {
+   bool neg = false;
+
+   if (**str == '-') {
+      neg = true;
+      (*str)++;
+   }
+
+   unsigned long long res = fast_strtoull_dec(str, maxlen);
+   assert(res <= LONG_LONG_MAX);
```

Comment:
```suggestion
   assert(res <= LLONG_MAX);
```

And I'm not sure what you are going to do if the input is exactly `-9223372036854775808`, since this input become an undefined behavior in the standard's sense.

Reply:
This won't matter, as this function would `assert` in that case.